### PR TITLE
Added protocol-interactiveselenium.jar dependency.

### PR DESCRIPTION
**Pull request by raghavbharadwaj**
Added LoginHandler22.class as example

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+#*.jar
```
Comment (chrismattmann) [inline .gitignore]: this should be added back. We should have @lewismc or @sujen1412 publish the protocol-interactiveselenium on the Central repo

File: nutch/selenium-handlers/src/main/java/gov/nasa/jpl/memex/nutch/protocol/selenium/handlers/login/LoginHandler22.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package gov.nasa.jpl.memex.nutch.protocol.selenium.handlers.login;
```
Comment (chrismattmann) [inline nutch/selenium-handlers/src/main/java/gov/nasa/jpl/memex/nutch/protocol/selenium/handlers/login/LoginHandler22.java]: Apache license, version 2 header

**Comment by chrismattmann** [comment]
where's the rest of the handlers?

**Comment by chrismattmann** [comment]
@raghavbharadwaj

File: .gitignore (hunk 1 above)
Comment (raghavbharadwaj) [inline .gitignore]: Until that happens, can we have the jar sitting locally (in the project) in lib? Helps me have code that compiles (or nearly does) on the repo.

Comment (chrismattmann) [inline .gitignore]: how about we simply add it using a local file maven install, like so:
```bash
mvn install:install-file \
-Dfile=$1 \
-DgroupId=org.apache.nutch -DartifactId=protocol-interactiveselenium -Dversion=1.12 \
-Dpackaging=jar -DgeneratePom=true
```

Comment (raghavbharadwaj) [inline .gitignore]: I understand Professor @chrismattmann but I don't understand what would be the location of the jar? Are you proposing we would not have the jar in the project repo, even until it is published on the Central repo?

**Comment by chrismattmann** [comment]
great work now we can incrementally update it. Thanks @raghavbharadwaj cc @sujen1412

**Comment by raghavbharadwaj** [comment]
Thanks Professor. I think MultiplePattern33.java (team 33) and LoginHandler36.java (team 36) are kind of well written and we could further refactor them. Can you take a look at them?

**Comment by chrismattmann** [comment]
sure I will take a look @raghavbharadwaj

**Pull request merged**